app/Jobs/RefreshPolyline.php

```diff
@@ -2,26 +2,49 @@
 
 namespace App\Jobs;
 
-use App\Http\Controllers\Backend\BrouterController;
+use App\Http\Controllers\ReRoutingController;
 use App\Models\Trip;
+use Cache;
 use Illuminate\Bus\Queueable;
 use Illuminate\Contracts\Queue\ShouldQueue;
 use Illuminate\Foundation\Bus\Dispatchable;
 use Illuminate\Queue\InteractsWithQueue;
 use Illuminate\Queue\SerializesModels;
+use Illuminate\Support\Facades\Log;
 use romanzipp\QueueMonitor\Traits\IsMonitored;
 
 class RefreshPolyline implements ShouldQueue
```

Comment:
Please skip this task if a polyline is specified in the schedule, as otherwise we will destroy the correct data again.